Week2/num4.js

```diff
@@ -0,0 +1,12 @@
+// 4
+let y = "Abdullah is doing homework";
+// 4.1
+console.log("the value of variabl y will be : Abdullah is doing the homework")
```

Comment:
Here you wrote:
"Abdullah is doing **the** homework"

But I think the value is:
"Abdullah is doing homework".

It's a small thing!

Reply:
And you forgot to add a ';' at the end of this line (which you seem to have forgotten at all `console.log` statements